examples/eks-getting-started/vpc.tf

```diff
@@ -49,6 +49,6 @@ resource "aws_route_table" "demo" {
 resource "aws_route_table_association" "demo" {
   count = 2
 
-  subnet_id      = aws_subnet.demo.*.id[count.index]
+  subnet_id      = aws_subnet.demo[count.index].id
```

Comment:
The legacy splatt expression effectively evaluated to:

```terraform
  subnet_id      = [for o in aws_subnet.demo: o.id ][count.index]
```

which is why the placement of `count.index` is shifted here.